examples/kafka/subscriber/wait-for-it.js

```diff
@@ -0,0 +1,17 @@
+const kafka = require('kafka-node');
+const client = new kafka.KafkaClient({ kafkaHost: process.env.ENVIRONMENT === 'local' ? process.env.INTERNAL_KAFKA_ADDR : process.env.EXTERNAL_KAFKA_ADDR });
+const Admin = kafka.Admin;
+const child_process = require('child_process');
+
+const admin = new Admin(client)
+const interval_id = setInterval(() => {
+  admin.listTopics((err, res) => {
+    if (res[1].metadata.architect) {
+      console.log('Kafka topic created');
+      clearInterval(interval_id);
+      child_process.execSync('npm start', { stdio: 'inherit' });
+    } else {
+      console.log('Waiting for Kafka topic to be created');
+    }
+  });
+}, 1000);
```

Comment:
Just fyi, this may have been more intuitive as a standalone script that exits zero or one for completion or failure. That would have allowed it to be generalized to something like this in the `CMD` field:

```sh
wait-for-kafka.sh && npm start
```

Hardcoding `npm start` into that script would make it harder to find where the actual start script is being called for editing and management. That being said, this is just an example so I'll approve since it all works locally now.